<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Luo Siyou</author>
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="StartButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StopButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>数据显示方式</string>
   </property>
   <widget class="QRadioButton" name="DisInWaveradioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>wave</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="DisInValueradioButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>value</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="DTRcheckBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DTR</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="RTScheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>RTS</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TransmittextEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearReceiveBufpushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveReceiveBufpushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearTransmitBufpushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendTransmitBufpushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="ReceiveDisInHEXradioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="ReceiveDisInCHARradioButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CHAR</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>520</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="TransmitDisInHEXradioButtonradioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="TransmitDisInCHARradioButtonradioButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CHAR</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>451</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTextBrowser" name="ReceiveBuftextBrowser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>451</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ReceiveZoneLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>接收数据值</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>接收数据波形</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="GraphicsWindow" name="WavegraphicsView"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>201</y>
     <width>71</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="OnOfflabel">
      <property name="text">
       <string>开/关</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SerialPortlabel">
      <property name="text">
       <string>串口号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Baudratelabel">
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Databitslabel">
      <property name="text">
       <string>数据位</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Stopbitslabel">
      <property name="text">
       <string>停止位</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Paritybitlabel">
      <property name="text">
       <string>校验位</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="TransmitzoneLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>发送区</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>121</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QComboBox" name="BaudratecomboBox">
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="DatabitscomboBox">
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="ParitybitcomboBox">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mark</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Space</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="OnOffButton">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="SerialPortcomboBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="StopbitscomboBox">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsWindow</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
